connector/connectionProperties.js

```diff
@@ -2,9 +2,8 @@
     var props = {};
     props["user"] = attr[connectionHelper.attributeUsername];
     props["password"] = attr[connectionHelper.attributePassword];
+    props["database"] = attr[connectionHelper.attributeDatabase];
     props["dialect"] = "mongosql";
-    props["database"] = attr[connectionHelper.attributeDatabase]
-    props["authSource"] = "admin";
 
     if (attr[connectionHelper.attributeSSLMode] == "require") {
         props["ssl"] = "true";
```

Comment:
Are SSL properties currently handled at all by the JDBC driver? I don't see any tests asserting that we handle them in `MongoDriverTest.java`. Beyond that, I think we also have additional questions to answer around how to handle conflicts between SSL options in the properties and in the URI.

If there's not a clear and obvious answer to these things at the moment, I would suggest that we remove this for now and create a new design doc section to discuss, so that this question doesn't hold up merging the rest of the PR

Reply:
I see that the `ssl` property gets added to the ConnectionString in the driver when I print the connection string.  I have also noticed that whether that property is set or not affects whether the driver can connect to ADL, with the local ADL instance it only works if unset.

I think one way to handle this can be to remove the SSL UI element as well and have the users specify it in the URI, what do you think?